docs/data/sql_functions.yml

```diff
@@ -617,6 +617,15 @@ collection:
   - sql: ARRAY_DISTINCT(haystack)
     table: haystack.arrayDistinct()
     description: Returns an array with unique elements. If the array itself is null, the function will return null. Keeps ordering of elements.
+  - sql: MAP_KEYS(map)
+    table: MAP.mapKeys()
+    description: Returns the keys of the map as array.
+  - sql: MAP_VALUES(map)
```

Comment:
Returns an unordered array containing the keys of the map like spark https://spark.apache.org/docs/latest/api/sql/index.html#map_keys
i suggest add unordered array

Reply:
This is just a special case. the spark logical is same. but it has unordered description

Reply:
From my point of Spark's description is not precise enough...
I added `No order guaranteed.`